### PR TITLE
Maintenance work

### LINKED ISSUE #139: node version too old in pipeline :(

### PR DESCRIPTION
**Pull request by k4y4k**
Tracked in [Story 14365](https://dev.azure.com/acrrm/CRM/_workitems/edit/14365).

Closes #139.

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
```
Comment (caitlinwillmot) [inline renovate.json:2]: Is it okay to introduce renovate for this project? Will we get flooded with PRs?

File: src/theme/SearchBar/lib/DocSearch.js [hunk 2]
```diff
@@ -16,291 +16,290 @@ import $ from "autocomplete.js/zepto";
...
+    autocompleteOptions = {
+      debug: false,
+      hint: false,
+      autoselect: true,
+    },
+    transformData = false,
+    queryHook = false,
+    handleSelected = false,
+    enhancedSearchInput = false,
+    layout = "collumns",
```
Comment (caitlinwillmot) [inline src/theme/SearchBar/lib/DocSearch.js:34]: Is this columns meant to be spelled like this?

File: renovate.json (hunk 1 above)
Comment (caitlinwillmot) [inline renovate.json:2]: Is it okay to introduce renovate for this project? Will we get flooded with PRs?

File: src/theme/SearchBar/lib/DocSearch.js [hunk 3]
```diff
@@ -16,291 +16,290 @@ import $ from "autocomplete.js/zepto";
...
+    }
+    this.autocomplete = autocomplete(this.input, autocompleteOptions, [
+      {
+        source: this.getAutocompleteSource(transformData, queryHook),
+        templates: {
+          suggestion: DocSearch.getSuggestionTemplate(this.isSimpleLayout),
+          footer: templates.footer,
+          empty: DocSearch.getEmptyTemplate(),
+        },
+      },
```
Comment (caitlinwillmot) [inline src/theme/SearchBar/lib/DocSearch.js:71]: Are trailing commas okay? Is this a lint or prettier rule?

File: static/admin/index.html [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
+    <script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Content Manager</title>
+  </head>
+  <body>
+    <!-- Include the script that builds the page and powers Netlify CMS -->
+    <script src="https://unpkg.com/netlify-cms@^2.0.0/dist/netlify-cms.js"></script>
+    <script>
+      CMS.registerPreviewStyle("archiaregular.css");
```
Comment (caitlinwillmot) [inline static/admin/index.html:13]: Good idea 👍

**Review by caitlinwillmot — CHANGES_REQUESTED** [review]
Thanks for cleaning this up Kayak, just a couple of questions if you could please review, thanks

File: README.md [hunk 5]
```diff
@@ -1,43 +1 @@
-# Website
-
```
Comment (fgolwala) [inline README.md:2]: Why was this cotent removed?

**Review by fgolwala — APPROVED** [review]

**Review by caitlinwillmot — APPROVED** [review]

**Comment by TJMatheson** [comment]
Very nice!

**Pull request merged**